util/notify/notify.asd

```diff
@@ -0,0 +1,13 @@
+;;;; notify.asd
+
+(asdf:compute-source-registry)
```

Comment:
What is the purpose of this line? Unless there is a good reason it should be removed